example/android/app/build.gradle

```diff
@@ -99,9 +99,8 @@ android {
 
     defaultConfig {
         applicationId "com.googlesigninsampleapp"
-        compileSdkVersion rootProject.ext.compileSdkVersion
-        buildToolsVersion rootProject.ext.buildToolsVersion
         minSdkVersion rootProject.ext.minSdkVersion
+        targetSdkVersion rootProject.ext.targetSdkVersion
```

Comment:
@jozan I removed compileSdkVersion and buildToolsVersion since they are in the block above. In this block, there should be minSdkVersion(you added that) and targetSdkVersion